### PR TITLE
cross platform compatible

### LINKED ISSUE #13: path problem in windows
need to use join instead of hard coded string

### PR DESCRIPTION
**Pull request by Arnab-Shanta-Anu**
this commit fixes #13 windows path problem

### CODE REVIEW

**Review by Arnab-Shanta-Anu — COMMENTED** [review]
using makedirs instead of mkdir and getting compatibility by os.path.join

**Comment by Arnab-Shanta-Anu** [comment]
@bijoy26 check it please

File: download-script.py [hunk 1]
```diff
@@ -13,14 +14,33 @@
     print("no user in that name found\nRun program again")
     exit(1)
 
+seperator = ""
+if (platform.system() == "Linux"):
+    seperator = "/"
+
+elif(platform.system() == "Windows"):
```
Comment (bijoy26) [inline download-script.py:21]: Seperators, this is one of the reasons the file systems are different and required different path formats.

**Comment by bijoy26** [comment]
> @bijoy26 check it please

I think latest _.exe update_ haven't been released yet.

**Comment by bijoy26** [comment]
@Arnab-Shanta-Anu tested the latest executable.
Both custom path and custom directory name work just fine.

**Comment by Arnab-Shanta-Anu** [comment]
thanks for the feedback

**Pull request merged**